### DISCUSSION TITLE
locked out in the github. Unable to change the password.

### DISCUSSION THREAD

**Discussion by Faraz31613** [community/community#42268] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Unable to change the password.

### Body

I don't remember my account's password. I can't change my account's password because two factor authentication is enabled and i changed my phone so don't have the authentication app working for my account and i also don't have recovery codes.
What should i do to recover my account.

**Comment by radar-blip** [comment]
You can try [Support](https://support.github.com/request)

**Comment by byrneh** [comment]
Your options are explained here
[recovering-your-account-if-you-lose-your-2fa-credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials)
